src/json-editor.component.html

```diff
@@ -22,7 +27,7 @@
   </div>
 
   <div id="middle-main-container" class="middle main-container" [class.compact]="config.compact" [shortcuts]="customShortcutKeys"
-    [ngClass]="isPreviewerHidden ? 'maximizeEditor' : 'maximizeEditor'">
+    [ngClass]="isPreviewerHidden ? 'maximizeEditor' : '' ">
```

Comment:
since you are fixing this it's better to use single class to boolean style biding. After some renaming something like  `[class.editor-fullscreen]="isPreviewHidden"` would be better.